### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by orduno**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | # |
| Primary OS tested on | Ubuntu 18.04 |
| Robotic platform tested on | Gazebo simulation of TB3 |

---

## Description of contribution in a few bullet points

CI broken due to linter fails, this PR corrects them.

### CODE REVIEW

File: nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright (c) 2019 Intel Corporation
```
Comment (SteveMacenski) [inline nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp]: This was a port from Nav1 https://github.com/ros-planning/navigation/blob/melodic-devel/costmap_2d/include/costmap_2d/testing_helper.h

This should probably still have an original copyright header added if you're also going to add intels

Comment (bpwilcox) [reply nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp]: There is no copyright header in the original

Comment (orduno) [reply nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp]: In that case, do keep the file copyright-less and suppress the lint fail?

@crdelsey do you have a suggestion on what to do in this case?

Comment (SteveMacenski) [reply nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp]: Even without the copyright header, they still own the copyright itself. Adding the header without including the original copyright holder is basically claiming copyright over their work, which is pretty unambiguously infringing. The idea of a copyright header doesn't assert a copyright, its more of a "reminder" that isn't strictly required for any particular legal reason 

I'm just suggesting adding this up top as well 
` // Copyright (c) Willow Garage `

Comment (SteveMacenski) [reply nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp]: But I approved it, not blocking over this

**Pull request merged**